### ISSUE TITLE
List rune balances in alphabetical order

### ISSUE THREAD

**Issue by gmart7t2** [ordinals/ord#4102]
Currently rune balances are sorted first by rune name length and then alphabetically which makes it hard to scan through for specific rune names.

Wouldn't it be better to sort strictly alphabetically? 

Here's an example. Shorter names come before longer names:
![Screenshot_20241125_142818_Presearch](https://github.com/user-attachments/assets/5725590a-e104-4547-9b0f-32c0bc32f876)

**Comment by raphjaph** [comment]
Yeah we could definitely sort those. Should be a one line change. You want to open a PR for that?
